javascript/javascript3/week2/promiseSettime.js

```diff
@@ -0,0 +1,13 @@
+afterSetTime(8).then(() => {
+  console.log("I am called asynchronously"); // logged out after 8 seconds
+});
+
+async function afterSetTime(resolveAfter) {
+  const setTimer = resolveAfter * 1000;
+  console.log(`Let us see.... in 8 sec. from now on`);
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
```

Comment:
there is a small mistake here. `resolve(..)` expects an argument. For example `resolve('timeout triggered after x seconds')`
For more details, see the following error message:
```
Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)
```